src/controllers/controller.h

```diff
@@ -73,8 +73,9 @@ class Controller : public QObject, ConstControllerPresetVisitor {
     // preset, not a pointer to the preset itself.
     void presetLoaded(ControllerPresetPointer pPreset);
 
-  // Making these slots protected/private ensures that other parts of Mixxx can
-  // only signal them which allows us to use no locks.
+    // Making these slots protected/private ensures that other parts of Mixxx
+    // can
+    // only signal them which allows us to use no locks.
```

Comment:
LOL. I guess we'd have to accept the fact that comments would just get screwed up. At least we humans can handle fuzzed data much better than complilers can. :)